tests/resources/Engine.py

```diff
@@ -3159,3 +3248,23 @@ def ctn_get_service_command(host_id: int, service_id: int):
         logger.console(
             f"Unable to find the command id of service ({host_id};{service_id})")
         return None
+
+def ctn_set_service_state(id: int, state: int):
```

Comment:
the description function has a standard. Look at others functions for examples.

Reply:
I'm almost sure there was already a function to do this job...

Reply:
yes the tests pass without it so i remove it